src/NuGet.CommandLine.StagingExtensions/StageManagementResource.cs

```diff
@@ -154,8 +252,14 @@ public StageManagementResource(string stageServiceUri, HttpSource httpSource)
                 {
                     string serverMessage = await response.Content.ReadAsStringAsync();
 
-                    throw new HttpRequestException(
-                        $"Response status code does not indicate success: {response.StatusCode}. Message: {serverMessage}");
+                    var errorMessage = $"Response status code does not indicate success: {response.StatusCode}.";

```

Comment:
It should be localized, it just hasn't happened yet in all of the code that was ported over. 
https://github.com/NuGet/Home/issues/2041